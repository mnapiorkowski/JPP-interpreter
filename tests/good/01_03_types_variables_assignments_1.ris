void main():
    let int a;
    println a;

    a = 42;
    println a;

    a++;
    println a;

    a--;
    println a;

    let bool b;
    println b;
    
    b = true;
    println b;
    
    let string c = "foo";
    println c;

    return;