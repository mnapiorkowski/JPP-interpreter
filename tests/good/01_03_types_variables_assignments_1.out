0
42
43
42
false
true
foo
